doc/en/developer/source/policies/community-process.rst

```diff
@@ -12,116 +12,58 @@ The GeoServer community process adheres to the following guidelines to ensure
 that it retains open development practices:
 
 * **Transparency** 
-  Project decisions are made in an open and transparent way. When a decision is
-  made it is clear as to why it was made.
+  Project decisions are made in an open and transparent way. Key decisions are presented *GeoServer Improvement Proposals* (:ref:`GSIP <gsip>`), discussed in public. When a decision is made it is clear as to why it was made.
```

Comment:
Ah, that's correct.. but the wiki is also git managed, they cannot make a pull request, but technically they can still play this way: http://roman-ivanov.blogspot.it/2013/11/how-to-merge-github-wiki-changes-from.html
It's quite convoluted, not sure we can enough to describe that approach.